README.md

```diff
@@ -231,15 +225,15 @@ async fn transaction_handler_name(
     // and the global state. It is parametrized by the
     // app state and the transaction context type, but the context is optional,
     // and defaults to the unit type.
-    context: TransactionHandlerContext<YOUR_STATE, YOUR_TRANSACTION_CONTEXT_TYPE>,
+    context: TransactionHandlerContext<YOUR_STATE>,
 ) -> Result<(), TransactionHandlerError> {
     // Do something like start a database transaction
     let mut transaction_context = TransactionContext { tx: start_transaction() }
 
     // Handle the events inside the incoming transaction.
     // We provide a simple method for this.
     context
-        .transaction
+        .event_processor
```

Comment:
I updated this to a new custom struct instead of being on the transaction struct, as this allows the TransactionStreamProcessor to inject some values into it for control, and it would not make sense to put that inside the actual transaction struct.